### DISCUSSION TITLE
Hide repeatable orders toggle

### DISCUSSION THREAD

**Discussion by mifriis** [DFHack/dfhack#2873] [Ideas]
With the new tools to work orders in 50.07-alpha1 i couldn't help but think that would be great for also hiding away those workorders that just take care of themself.

I have a full page of automation setup and need to scroll down past them to see the single order ones.

Could this be a nifty QoL feature perhaps?

![image](https://user-images.githubusercontent.com/1443594/218259991-071b86ec-c516-4e8c-bfb6-63d00914aee7.png)
and with a "ctrl+h" 
![image](https://user-images.githubusercontent.com/1443594/218260034-f1eabe11-a6d2-43f5-b333-575cdba3512b.png)

ready for a quick order

**Comment by omgpointless** [comment]
while not the solution you suggested, a temporary work around could be to utilize the orders script, and the sort functionality in particular.
On my forts I have this repeat added:
`repeat --name orders-sort --time 1 --timeUnits days --command [ orders sort ]`
this will put any one-time orders at the very top.
